docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+</Admonition>
+
+#### Example Slack request
+
+<video controls>
+  <source src="../../../img/enterprise/plugins/slack/slack.mp4" type="video/mp4" />
+  <source src="../../../img/enterprise/plugins/slack/slack.webm" type="video/webm" />
+  Your browser does not support the video tag.
+</video>
+
+## Prerequisites
+
+This guide assumes that you have:
+
+- Teleport (=teleport.version=).
+- Admin privileges with access to [`tctl`](https://gravitational.com/teleport/docs/cli-docs/#tctl).
+- Slack Admin Privileges to create an app and install it to your workspace.
+
+<Admonition type="Tip" title="tip">
+  Read [Getting Started](../../getting-started.mdx) to install Teleport and configure it with a user and role.
+</Admonition>
+
+## Step 1/4. Create a user and role for access
+
+1. Log into your Teleport Authentication Node. The Authentication Node is where you'll most likely perform administrative tasks and run the powerful `tctl` ("tea-kettle") admin tool. 
+
+2. Create a new user and role with only API access to the `access_request` API. 
+  
+   The following YAML file specifies a non-interactive bot `access-plugin` user and role.
+
+   ```yaml
+   kind: user
+   metadata:
+     name: access-plugin
+   spec:
+     roles: ['access-plugin']
+   version: v2
+   ---
+   kind: role
+   version: v3
+   metadata:
+     name: access-plugin
+   spec:
+     allow:
+       rules:
+         - resources: ['access_request']
+           verbs: ['list', 'read']
+         - resources: ['access_plugin_data']
+           verbs: ['update']
+       # Teleport currently refuses to issue certs for a user with 0 logins.
+       # This restriction may be lifted in future versions.
+       logins: ['access-plugin-not-used']
+   ```
+    
+   Name the YAML file something like `access.yaml`.
+  
+   Ensure that the role and name is `access-plugin`; the plugin requires this to function properly.
+
+   <Admonition type="warning" title="Warning">
+    The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in customer-facing production environments per the *Principle of Least Privilege* (POLP). 
+   </Admonition>
+
+3. Create the bot for Teleport using the created role:
+  
+   ```bash
+   sudo tctl create -f access.yaml
+   ```
+
+<Admonition type="tip">
+  If you're using other plugins, you'll likely want to create users and roles for each plugin. It's generally good practice to separate users and roles by concern.
+</Admonition>
+
+## Step 2/4. Export access-plugin certificate
+
+Teleport Plugin uses the `access-plugin` role and user to perform the approval. 
+
+We export the identity files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
+
+```bash
+sudo tctl auth --auth-server=auth.example.com:3025 sign --format=tls --user=access-plugin --out=auth --ttl=1h
```

Comment:
You only need sudo if running locally, with cloud the sudo is not necessary

Reply:
Let's also switch the instruction to use `tctl auth sign --format=file --auth-server=proxy.example.com:443`. This is better, because it generates only one file, works both with cloud and self-hosted editions, does not require sudo

Reply:
You know, I left this in here despite the previous consensus since non of the Access Controls guides had this change made. Will remove. Thanks!

Reply:
I've tried: `sudo tctl auth sign --format=file --out=auth --auth-server=slacktest.go-teleport.com:443` and get:

```
INFO [CLIENT]    "Cannot connect to the auth server: failed direct dial to auth server: Get \"https://teleport.cluster.local/v2/configuration/name\": remote error: tls: internal error\n\tGet \"https://teleport.cluster.local/v2/configuration/name\": remote error: tls: internal error.\nIs the auth server running on \"slacktest.go-teleport.com:443\"?" utils/cli.go:235
[CLIENT]       Cannot connect to the auth server: failed direct dial to auth server: Get "https://teleport.cluster.local/v2/configuration/name": remote error: tls: internal error
	Get "https://teleport.cluster.local/v2/configuration/name": remote error: tls: internal error.
Is the auth server running on "slacktest.go-teleport.com:443"?
```

I believe I previously tried this and encountered the same thing:

1. https://github.com/gravitational/teleport-plugins/issues/230
2. https://github.com/gravitational/teleport-plugins/issues/239

Is some part of that command I used wrong? Thanks! I'll happily update the command and change the TOML file - also: https://github.com/gravitational/teleport-plugins/pull/259 - those need to be updated and if there's an easier way to pass an identity file, the better!

Reply:
Corrected the command and also significantly refactored the steps. I moved all of the Slack config to the beginning, then all of the Teleport steps thereafter.